test/with_qiskit/test_circuit_assert_equal.py

```diff
@@ -14,224 +14,64 @@ class TestCircuitAssertEqual(unittest.TestCase):
     {Your directory where you git-cloned quantestpy}/quantestpy
-    $ python -m unittest test.test_circuit_assert_equal
+    $ python -m unittest test.with_qiskit.test_circuit_assert_equal
     ...
     ----------------------------------------------------------------------
     Ran 3 tests in 0.006s
 
     OK
     """
 
-    def test_msg_from_miss_all_info_for_circuit_a(self,):
-        test_circuit = TestCircuit(2)
+    def test_msg_from_wrong_input_type_for_circuit_a(self,):
+
+        circuit_a = np.array([[0, 1], [1, 0]])  # wrong
+        circuit_b = TestCircuit(2)  # correct
 
         try:
             self.assertIsNotNone(
                 circuit.assert_equal(
-                    test_circuit_b=test_circuit
+                    circuit_a=circuit_a,
+                    circuit_b=circuit_b
                 )
             )
 
         except QuantestPyError as e:
 
-            expected_error_msg = "quantestpy.exceptions.QuantestPyError: " \
-                + "Missing information for circuit A. " \
-                + "One of the following must be given: " \
-                + "qasm_a, qiskit_circuit_a and test_circuit_a.\n"
+            expected_error_msg = \
+                "quantestpy.exceptions.QuantestPyError: " \
+                + "circuit_a must be one of the following: " \
+                + "qasm, qiskit.QuantumCircuit and TestCircuit.\n"
 
-            actual_error_msg = traceback.format_exception_only(type(e), e)[0]
+            actual_error_msg = \
+                traceback.format_exception_only(type(e), e)[0]
 
             self.assertEqual(expected_error_msg, actual_error_msg)
 
-    def test_msg_from_miss_all_info_for_circuit_b(self,):
-        test_circuit = TestCircuit(3)
+    def test_msg_from_wrong_input_type_for_circuit_b(self,):
+
+        circuit_a = QuantumCircuit(3)  # correct
+        circuit_b = [[0, 1], [1, 0]]  # wrong
 
         try:
             self.assertIsNotNone(
                 circuit.assert_equal(
-                    test_circuit_a=test_circuit
+                    circuit_a=circuit_a,
+                    circuit_b=circuit_b
                 )
             )
 
         except QuantestPyError as e:
 
-            expected_error_msg = "quantestpy.exceptions.QuantestPyError: " \
-                + "Missing information for circuit B. " \
-                + "One of the following must be given: " \
-                + "qasm_b, qiskit_circuit_b and test_circuit_b.\n"
+            expected_error_msg = \
+                "quantestpy.exceptions.QuantestPyError: " \
+                + "circuit_b must be one of the following: " \
+                + "qasm, qiskit.QuantumCircuit and TestCircuit.\n"
 
-            actual_error_msg = traceback.format_exception_only(type(e), e)[0]
+            actual_error_msg = \
+                traceback.format_exception_only(type(e), e)[0]
 
             self.assertEqual(expected_error_msg, actual_error_msg)
 
-    def test_msg_from_too_much_info_for_circuit_a(self,):
```

Comment:
The following 4 tests are deleted because the corresponding error messages are removed from the source code.